examples/rules.mk

```diff
@@ -170,6 +170,12 @@ flash-pyocd: $(BUILD)/$(PROJECT).hex
 	pyocd flash -t $(PYOCD_TARGET) $<
 	pyocd reset -t $(PYOCD_TARGET)
 
+# flash with Black Magic Probe
+flash-bmp: $(BUILD)/$(PROJECT).elf
+	$(GDB) --batch -ex 'target extended-remote /dev/ttyBmpGdb' -ex 'monitor swdp_scan' -ex 'attach 1' -ex load  $<
```

Comment:
does the `/dev/ttyBmpGdb` name changes depending on the os (windows/macos/linux) like CDC Serial. If it does we may need to have user to specify to `SERIAL=` or `BMP=` similar to other dfu via serial. Of course, we could have with default value that works on most platform with `BMP ?=`

Reply:
Yes, it'll be different on different systems. I'll add the BMP variable as you suggested.